index.js

```diff
@@ -1,4 +1,52 @@
 "use strict";
+
+import { cardGeneration } from "./Deck.js";
+import Mix from "./Mix.js";
+import Player from "./Player.js";
+import distribuite from "./Distribue.js";
```

Comment:
There is a typo here 
```suggestion
import distribuite from "./Distribuite.js";
```